bibliography.bib

```diff
@@ -1,3 +1,14 @@
+@misc{jin_whan_bae_2017_854820,
+	author       = {Jin Whan Bae and
+	gyutaepark and
```

Comment:
Is @gyutaepark's name one word? I think it's three. This is a real citation and should be configured in zenodo with real human names.